api/views/phones.py

```diff
@@ -445,10 +444,18 @@ def inbound_sms(request):
     if inbound_from == real_phone.number:
         _handle_sms_reply(relay_number, real_phone, inbound_body)
         return response.Response(
-            status=200, data={"message": "Sent reply to last sender."}
+            {"message": "Sent reply to last sender."},
+            status=200,
+            template_name="twiml_empty_response.xml",
         )
 
-    _check_disabled(relay_number, "texts")
+    number_disabled = _check_disabled(relay_number, "texts")
+    if number_disabled:
+        say = "Sorry, that number is not accepting texts."
+        return response.Response(
+            {"say": say}, status=200, template_name="twiml_blocked.xml"
+        )
```

Comment:
This is OK but misleading. The `<Say>` command does nothing for incoming text messages, so there is no response. What about the `twiml_empty_response.xml` instead?